README.md

```diff
@@ -0,0 +1,63 @@
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet) ![Ruby](https://img.shields.io/badge/-Ruby-red)
+
+# Catalog of my things
+
+> This project is a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files but you will also prepare a database with tables structure analogical to your program's class structure.
+>
+![Catalog of my things UML class diagram](catalog_of_my_things.png)
+
+## Video Presentation
+[![Presentation](https://res.cloudinary.com/marcomontalbano/image/upload/v1668818126/video_to_markdown/images/google-drive--14zVCP5ZjpNqarxXBbu5BRn36_miZM4of-c05b58ac6eb4c4700831b2b3070cd403.jpg)](https://drive.google.com/file/d/14zVCP5ZjpNqarxXBbu5BRn36_miZM4of/view "Presentation")
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Follow the console instruction
+
+### Setup and run
+
+- Open the terminal on your computer
+- Navigate through directories using [change directory (cd)](https://www.howtogeek.com/659411/how-to-change-directories-in-command-prompt-on-windows-10) command to where you want to clone the repository.
+
+- _Install dependencies locally_
+  - bundle install
+
+- run
+  - ruby main.rb
+
```

Comment:
- [OPTIONAL] Your README is well documented 💯 I just suggest adding another command for testing 👍
 ```bash
 rspec .\spec
```